src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -4,15 +4,67 @@
 import javafx.geometry.Pos;
 import javafx.scene.control.Label;
 
+import javafx.scene.control.Tooltip;
 import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.StandardField;
 
 public class FieldNameLabel extends Label {
+    private Tooltip tip;
+    private static final String AUTHOR = "author";
+    private static final String BIBTEXKEY = "bibtexkey";
+    private static final String JOURNAL = "journal";
+    private static final String TITLE = "title";
+    private static final String YEAR = "year";
 
-    public FieldNameLabel(Field field) {
+    private static final String AUTHOR_TIP = "Multiple authors separated with 'and', e.g. author1 and author2";
+    private static final String BIBTEXKEY_TIP = "[First author'last name][Article year] e.g. Jones2020";
+    private static final String JOURNAL_TIP = "The name of the journal";
```

Comment:
These tooltips should be localized using `Localization.lang` and adding them to `JabRef_en.prop`.

Reply:
Yes, this works. Thank you!